pkg/cache/v3/simple.go

```diff
@@ -154,7 +150,7 @@ func NewSnapshotCacheWithHeartbeating(ctx context.Context, ads bool, hash NodeHa
 	return cache
 }
 
-func (cache *snapshotCache) sendHeartbeats(ctx context.Context, node string) {
+func (cache *snapshotCache) sendHeartbeats(_ context.Context, node string) {
```

Comment:
Were there linters complaining about this? I generally think this is bad to have in param naming schemes

Reply:
IIRC this is an unused variable warning. Perhaps I can just remove the parameter altogether.

Reply:
I'd remove the param